server/routes/login.router.js

```diff
@@ -0,0 +1,15 @@
+const { loginService } = require("../services/login.service");
+const { success, fail } = require("../utils/httpresult");
+
+const loginRouter = async (req, res) => {
+  const { email, password } = req.query;
```

Comment:
Once again, we'll definitely want to use the request body rather than query parameters. Having the plaintext password in the query string is a security risk.